ColorPicker/ColorPicker.js

```diff
@@ -240,8 +238,20 @@ const ColorPickerBase = kind({
 	}
 });
 
+// This handler is meant to accommodate using `ColorPicker`'s `onChange` prop from `Changeable`
+// as the `onSelect` handler for its `Group` component that lists the set of pre-defined color
+// choices.  `onSelect` sends the chosen value in the `data` prop of its event but `Changeable`
+// expects `value`.  The slider handlers for hue, saturation, and light values in `ColorPickerExtended`
+// use this handler to update the `value` prop via `onChange` as well.
+const colorChangeHandler = handle(
```

Comment:
Let's move this into the class below. It took me a while to figure out where `this.onChange` came from because it's the only handler defined outside of the class.